packages/garbo/src/tasks/embezzler.ts

```diff
@@ -0,0 +1,29 @@
+import { Quest } from "grimoire-kolmafia";
+import { $item, $items, $location, get, set } from "libram";
+import { GarboTask } from "./engine";
+import { GarboStrategy, Macro } from "../combat";
+import { getChangeLastAdvLocationMethod } from "../embezzler/lib";
+
+export const SetupEmbezzlerQuest: Quest<GarboTask> = {
+  name: "SetupEmbezzler",
+  tasks: [
+    {
+      // Need the daily dungeon to either be totally finished or to be on a NC we can walk away from
+      name: "Setup Daily Dungeon",
+      outfit: { equip: $items`ring of Detect Boring Doors` },
+      // walk away from any nc we can walk away from, skip boring doors, open the final chest
+      choices: () => ({ 689: 1, 690: 2, 691: 2, 692: 8, 693: 3 }),
+      acquire: [{ item: $item`ring of Detect Boring Doors` }],
+      ready: () => getChangeLastAdvLocationMethod() === "dailydungeon",
```

Comment:
```suggestion
      ready: () => getChangeLastAdvLocationMethod() === "dailydungeon" && have($item`miniature crystal ball`) && runningGregFights(),
```
or something like this